src/test/java/seedu/address/model/person/ContainsKeywordsPredicateTest.java

```diff
@@ -0,0 +1,254 @@
+package seedu.address.model.person;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.logic.parser.CliSyntax;
+import seedu.address.logic.parser.Prefix;
+import seedu.address.testutil.PersonBuilder;
+
+public class ContainsKeywordsPredicateTest {
+
+    @Test
+    public void equals() {
+        List<String> firstPredicateKeywordList = Collections.singletonList("first");
+        List<String> secondPredicateKeywordList = Arrays.asList("first", "second");
+
+        create_equals(
+                CliSyntax.PREFIX_NAME,
+                firstPredicateKeywordList,
+                secondPredicateKeywordList
+        );
+        create_equals(
+                CliSyntax.PREFIX_ADDRESS,
+                firstPredicateKeywordList,
+                secondPredicateKeywordList
+        );
+        create_equals(
+                CliSyntax.PREFIX_EMAIL,
+                firstPredicateKeywordList,
+                secondPredicateKeywordList
+        );
+        create_equals(
+                CliSyntax.PREFIX_TELEGRAM_HANDLE,
+                firstPredicateKeywordList,
+                secondPredicateKeywordList
+        );
+        create_equals(
+                CliSyntax.PREFIX_PHONE,
+                firstPredicateKeywordList,
+                secondPredicateKeywordList
+        );
+        create_equals(
+                CliSyntax.PREFIX_MODULE_TAG,
+                firstPredicateKeywordList,
+                secondPredicateKeywordList
+        );
+        create_equals(
+                CliSyntax.PREFIX_GROUP_TAG,
+                firstPredicateKeywordList,
+                secondPredicateKeywordList
+        );
```

Comment:
It does not look like standard Java if you create methods like these.

Reply:
Changed to camel case.